deploy/requirements.txt

```diff
@@ -1,4 +1,4 @@
-aws-cdk-lib==2.78.0
+aws-cdk-lib==2.99.0
```

Comment:
Could we also update the `backend/dataall/base/cdkproxy/requirements.txt` to have the same dependency for `aws-cdk-lib==2.99.0` 

I was able to deploy the code changes listed here as well as the change to `aws-cdk-lib==2.99.0`  in the other `requirements.txt` as well successfully but do you foresee any issues doing so @dlpzx @nikpodsh ?

I will test accessing the frontend and deploying a stack to ensure all still works as well - I think these code changes look good but could we point them to `v2m1m0` branch?

Reply:
I also tested creating/deploying the dataset stack - I would think be best to test create/delete of all the stack resources (i.e. env, notebook, ml studio, pipelines) if we do update the `aws-cdk-lib` requirement on cdkproxy as well 

This is something we would do holistically for validation testing on `v2m1m0` branch when ready for release if we do point this PR to that branch and add that additional code change but let me know your all thoughts when you get a chance

Reply:
Lastly confirmed no Lambda Runtimes using Node14 of Python3.7 after the PR code changes implemented

Reply:
I agree with updating `backend/dataall/base/cdkproxy/requirements.txt`. For testing all functionalities I would go for the final review in V2.1 validation. Otherwise it looks good to me

Reply:
Agreed, I missed that cdkproxy is also using it. Updated!